packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewer.spec.js

```diff
@@ -18,7 +18,7 @@ describe('Component > SingleImageViewer', function () {
     describe('with title', function () {
       it('should render the title', function () {
         render(<DefaultStory />)
-        const title = screen.getByText('Subject 1234')
+        const title = screen.getByTitle(`Subject ${subject.id}`)
```

Comment:
[`byTitle`](https://testing-library.com/docs/queries/bytitle/) queries find HTML elements with `title` attributes and SVG images containing `<title>`.